implementations/micrometer-registry-wavefront/src/main/java/io/micrometer/wavefront/WavefrontConfig.java

```diff
@@ -31,7 +31,7 @@
  * @author Jon Schneider
  * @since 1.0.0
  */
-public interface WavefrontConfig extends StepRegistryConfig {
+public interface WavefrontConfig extends PushRegistryConfig {
```

Comment:
@hanwavefront I'm curious what was the motivation for changing this from Step to Push? This is ending up breaking some tests in Spring Boot.

Reply:
Sorry for the silly question. After refreshing my memory on this change, it all makes sense now. The config type changed to match the change to the MeterRegistry type, in order to stop shipping rate-normalized metrics.